.gitignore

```diff
@@ -30,5 +30,9 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
+# Ignore config files
+/config/database.yml
```

Comment:
Why should changes to the database.yml not be checked into version control?

Reply:
Because it is a defacto standard to ignore the database.yml. Additionally, each deployment server has a specific database configuration, which is located in the shared files folder, which leads to conflicts, if there is another database.yml in the repository.